docs/_guide/community.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: guide
+title: Community
+slug: community
+---
+
+{::options toc_levels="1..3" /}
+* ToC
+{:toc}
+
+
+There are many great resources and locations to learn about LitElement and web components,
+share what you've built, and more. All community participation is subject to the Polymer Project's
+<a href="https://github.com/Polymer/project/blob/master/Code_of_Conduct.md">Code of Conduct</a>—be
+excellent to each other!
+
+*   <strong>Join us on <a href="https://polymer-slack.herokuapp.com/">Slack</a>!</strong> 
+    Polymer Slack has thousands of users discussing LitElement, web components, and related topics.
```

Comment:
Instead of "Polymer Slack", how about "the Polymer Project Slack workspace"?

Reply:
Or "the Polymer Project Slack"?

Reply:
Yeah. I was trying to enable find-in-page here, but I think the longer description is better & web search for "polymer slack" should find this just fine. Will update.

Reply:
Fixed this. Used the longer form. Also updated to the official slack invite link, same as the footer icon (had the old invite page here).